openshift-knative-operator/go.mod

```diff
@@ -3,29 +3,29 @@ module github.com/openshift-knative/serverless-operator/openshift-knative-operat
 go 1.14
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.7.1-0.20200907061046-05415f1de66d // indirect
-	github.com/census-instrumentation/opencensus-proto v0.3.0 // indirect
 	github.com/google/go-cmp v0.5.2
-	github.com/grpc-ecosystem/grpc-gateway v1.14.8 // indirect
-	github.com/manifestival/manifestival v0.6.1-0.20200803172850-17489fb53356
-	github.com/openshift/api v0.0.0-20200917102736-0a191b5b9bb0
-	k8s.io/api v0.18.8
+	github.com/google/uuid v1.1.2 // indirect
+	github.com/manifestival/manifestival v0.6.1
+	github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
+	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43 // indirect
+	golang.org/x/sys v0.0.0-20200915084602-288bc346aa39 // indirect
+	golang.org/x/tools v0.0.0-20200915031644-64986481280e // indirect
+	google.golang.org/genproto v0.0.0-20200914193844-75d14daec038 // indirect
+	google.golang.org/grpc v1.32.0 // indirect
+	k8s.io/api v0.19.0
 	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	k8s.io/code-generator v0.18.8 // indirect
-	knative.dev/operator v0.17.2
-	knative.dev/pkg v0.0.0-20200831162708-14fb2347fb77
-	knative.dev/test-infra v0.0.0-20200915193842-f4d4232c1f04
+	knative.dev/eventing v0.14.1-0.20200428210242-f355830c4d70 // indirect
```

Comment:
Although it is "indirect", do we need eventing v0.14.1? :thinking:

Reply:
Good catch, added eventing to the set of floating deps.